packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -62,22 +63,65 @@ export abstract class CosmosSdkBaseAdapter<T extends CosmosChainTypes> implement
   async getAccount(pubkey: string): Promise<chainAdapters.Account<T>> {
     try {
       const caip = this.getCaip2()
-      const { chain, network } = caip2.fromCAIP2(caip)
       const { data } = await this.providers.http.getAccount({ pubkey })
 
+      const delegations = (
+        data.delegations.length ? data.delegations : undefined
```

Comment:
Rather than dealing with a ternary here, if we require the type on the cosmos account, we can just do the map, and if there is no length, then it'll set it to an empty array.